src/test/java/io/debezium/connector/vitess/TestHelper.java

```diff
@@ -49,14 +49,15 @@ public class TestHelper {
     // Use the same username and password for vtgate and vtctld
     private static final String USERNAME = "vitess";
     private static final String PASSWORD = "vitess_password";
+    private static final String PASSWORD_FILE = "vitess_password_file.json";
```

Comment:
Maybe name it "vitess_auth_file.json" since it contains both username and password?

Reply:
Yes, I can rename the file name.  Initially I introduced the new config as 'database.password_file' so some of the test cases still use password_file.  Later I decide to use config name 'database.auth_file' but haven't updated the test file name yet.